demos/interactive_face_detection_demo/face.cpp

```diff
@@ -57,6 +62,10 @@ bool Face::isMale() {
     return _maleScore > _femaleScore;
 }
 
+bool Face::isSpoof() {
+    return _spoof_confidence <= 50.;
```

Comment:
If net's confidence that a face is spoof is low, then the face should be treated is real. You should name the method `isReal()` or negate the return value.